gravitee-apim-console-webui/package-lock.json

```diff
@@ -8775,9 +8775,9 @@
           }
         },
         "codemirror": {
-          "version": "5.63.3",
-          "resolved": "https://registry.npmjs.org/codemirror/-/codemirror-5.63.3.tgz",
-          "integrity": "sha512-1C+LELr+5grgJYqwZKqxrcbPsHFHapVaVAloBsFBASbpLnQqLw1U8yXJ3gT5D+rhxIiSpo+kTqN+hQ+9ialIXw=="
+          "version": "5.64.0",
+          "resolved": "https://registry.npmjs.org/codemirror/-/codemirror-5.64.0.tgz",
```

Comment:
Stange... ui-components use codemirror 5.63

Reply:
yes but the ui-components dependencies is `"codemirror": "^5.63.2",` so minor increment is ok, i think 🤔